File: sw-expert-academy/d1/c++/2072.cpp
#include <iostream>

using namespace std;

int main(int argc, char const *argv[])
{
    int test_case;
	int T;

    cin>>T;

    for(test_case = 1; test_case <= T; ++test_case)
	{
        int num, sum = 0;
        for (int i = 0; i < 10; i++) {
            cin >> num;
            if (num % 2 == 1) {
                sum += num;
            }
        }
        cout << "#" << test_case << " " << sum <<endl;
	}
    return 0;
}

File: sw-expert-academy/d1/python/2072.py
T = int(input())
for test_case in range(1, T+1):
    sum = 0
    num = list(map(int, input().split(" ")))
    for j in num:
        if (j % 2 == 1):
            sum += j
    print(f'#{test_case} {sum}')